packages/plugins/EVM/src/state/Connection/error.ts

```diff
@@ -47,8 +48,13 @@ export function getError(error: unknown, response?: JsonRpcResponse | null, fall
         ) {
             return 'Transaction was failed due to an internal JSON-RPC server error.'
         }
-        return internalError.message
+        return undefined
     })()
 
-    return new Error(internalErrorMessage)
+    if (recognizedErrorMessage) {
+        const error = new Error(recognizedErrorMessage) as RecognizedError
+        error.isRecognized = true
```

Comment:
`{...new Error(recognizedErrorMessage), isRecognized: true}`, this syntax is not adaptable for `Error`, it will only return `{isRecognized: true}`